rest/resource-server/src/docs/asciidoc/components.adoc

```diff
@@ -135,6 +135,33 @@ include::{snippets}/should_document_get_component_attachment_info/curl-request.a
 ===== Example response
 include::{snippets}/should_document_get_component_attachment_info/http-response.adoc[]
 
+[[resources-component-attachment-upload]]
+==== Upload attachment to component
+
+A `POST` request is used to upload attachment to component
+
+[red]#Request structure#
+|===
+|Path |Type |Description
+
+|file
+|NA
+|File path of the attachment
+
+|filename
+|String
+|Name of the file
+
+|attachmentContentId
+|String
+|Attachment content id
+|===
+
```

Comment:
Maybe add a section for 'Response structure' as well? Where you just say, that the complete project/component/release will be returned.

Reply:
Added the same for Project,Component and Release